app/main/index.ts

```diff
@@ -38,13 +38,17 @@ const mainURL = 'file://' + path.join(__dirname, '../renderer', 'main.html');
 
 const singleInstanceLock = app.requestSingleInstanceLock();
 if (singleInstanceLock) {
-	app.on('second-instance', () => {
+	app.on('second-instance', (event, argv) => {
 		if (mainWindow) {
 			if (mainWindow.isMinimized()) {
 				mainWindow.restore();
 			}
 
 			mainWindow.show();
+			if (process.platform !== 'darwin') {
+				const deepLinkingUrl = argv.slice(1);
+				handleDeepLink(deepLinkingUrl[(isDev && process.platform === 'win32') ? 1 : 0]);
```

Comment:
Might me good to mention that this function is for linux and windows (same as you have mentioned for macos below).